### PR TITLE
Create harvard-nus-geography.csl

### PR DESCRIPTION
**Pull request by EverLearner**
Citation style for Geography Honours Thesis in National University of Singapore.

### CODE REVIEW

File: harvard-nus-geography.csl [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+      </term>
+      <term name="editortranslator" form="verb">edited and translated by</term>
+      <term name="edition" form="short">ed.</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <names variable="container-author" delimiter=", " suffix=", ">
+          <name and="text" initialize-with=". " delimiter=", " sort-separator=", " name-as-sort-order="all"/>
```
Comment (adam3smith) [inline harvard-nus-geography.csl]: names should be with `and="symbol"` throughout. `name-as-sort-order="all"` is wrong here.

File: harvard-nus-geography.csl [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+        <names variable="editor translator" delimiter=". ">
+          <label form="verb" text-case="capitalize-first" suffix=" "/>
+          <name and="text" initialize-with=". " delimiter=", "/>
+        </names>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="symbol" delimiter-precedes-last="never" initialize-with=". " name-as-sort-order="all"/>
```
Comment (adam3smith) [inline harvard-nus-geography.csl]: `initialize-with="."` (no space)

File: harvard-nus-geography.csl [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+        <text term="in"/>
+      </if>
+    </choose>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year">
+    <sort>
+      <key macro="year-date"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter="; ">
+      <group delimiter=", ">
```
Comment (adam3smith) [inline harvard-nus-geography.csl]: should be `delimiter=":"`

File: harvard-nus-geography.csl [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+        <group delimiter=", ">
+          <text macro="author-short"/>
+          <text macro="year-date"/>
+        </group>
+        <group>
+          <label suffix=" " variable="locator" form="short"/>
```
Comment (adam3smith) [inline harvard-nus-geography.csl]: I don't think you want the label here

File: harvard-nus-geography.csl [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+          <group delimiter=", ">
+            <text macro="title"/>
+            <group delimiter=" ">
+              <text macro="container-prefix"/>
+              <text macro="editor"/>
+              <text macro="container-title"/>
+            </group>
+          </group>
+        </group>
+        <text macro="secondary-contributors"/>
```
Comment (adam3smith) [inline harvard-nus-geography.csl]: I think this will need a prefix

File: harvard-nus-geography.csl (hunk 1 above)
Comment (EverLearner) [reply harvard-nus-geography.csl]: Ditto the and="symbol". M

May I check with you why the `name-as-sort-order="all"` should be changed? All authors' names ought to be displayed as *lastname-firstname*.

File: harvard-nus-geography.csl (hunk 2 above)
Comment (EverLearner) [reply harvard-nus-geography.csl]: Agreed. Thanks!

File: harvard-nus-geography.csl (hunk 3 above)
Comment (EverLearner) [reply harvard-nus-geography.csl]: Agreed.

File: harvard-nus-geography.csl (hunk 4 above)
Comment (EverLearner) [reply harvard-nus-geography.csl]: Agreed. I missed this one out!

File: harvard-nus-geography.csl (hunk 5 above)
Comment (EverLearner) [reply harvard-nus-geography.csl]: I suppose it is `prefix=", "`

**Comment by rmzelle** [comment]
Thanks! (I renamed the style to how we typically do things here)

**Pull request merged**